src/drf_yasg/app_settings.py

```diff
@@ -33,6 +33,8 @@
     'DEFAULT_API_URL': None,
 
     'USE_SESSION_AUTH': True,
+    'CSRF_COOKIE_NAME': settings.CSRF_COOKIE_NAME,
```

Comment:
I'm pretty sure these will be available, but I can change this to `gettar` like below if that is preferred.